packages/field-base/src/input-mixin.js

```diff
@@ -51,6 +51,7 @@ export const InputMixin = dedupingMixin(
             value: '',
             observer: '_valueChanged',
             notify: true,
+            sync: true,
```

Comment:
Some tests started to fail, apparently because of this change:

```
packages/checkbox/test/checkbox-lit.test.js:

 ❌ checkbox > default > should toggle checked property on input click
      AssertionError: expected true to be false
      + expected - actual
      
      -true
      +false
      
      at Ka.<anonymous> (packages/checkbox/test/checkbox.common.js:[49](https://github.com/vaadin/web-components/actions/runs/6182725138/job/16783021878?pr=6487#step:5:50):37)

 ❌ checkbox > default > should toggle checked property on label click
      AssertionError: expected true to be false
      + expected - actual
      
      -true
      +false
      
      at Ka.<anonymous> (packages/checkbox/test/checkbox.common.js:[57](https://github.com/vaadin/web-components/actions/runs/6182725138/job/16783021878?pr=6487#step:5:58):37)
```

Reply:
Fixed them by adding `nextUpdate()`.